recipes-support/swupdate/swupdate_%.bbappend

```diff
@@ -19,7 +20,11 @@ SYSTEMD_SERVICE:${PN} += "swupdate-sysrestart.service"
 
 do_install:append () {
 	install -m 644 ${WORKDIR}/swupdate.cfg ${D}/${sysconfdir}
-	install -m 644 ${WORKDIR}/swu_public.pem ${D}/${sysconfdir}
+	if ${@bb.utils.contains('DISTRO', 'quilt-qsm-release quilt-donut-release', 'true', 'false', d)}; then
```

Comment:
Do we have access to 'DISTRO' from this recipe? Not sure how bitbake grabs recipes.

Reply:
Yes, but i should use contains_any()